### PR TITLE
Standardize the generation of certificates for karmadactl init

### PR DESCRIPTION
**Pull request by lonelyCZ**
Signed-off-by: lonelyCZ <531187475@qq.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
```
[root@master67 karmada]# tree
.
├── crds
│   ├── bases
│   │   ├── config.karmada.io_resourceinterpreterwebhookconfigurations.yaml
│   │   ├── multicluster.x-k8s.io_serviceexports.yaml
│   │   ├── multicluster.x-k8s.io_serviceimports.yaml
│   │   ├── networking.karmada.io_multiclusteringresses.yaml
│   │   ├── policy.karmada.io_clusteroverridepolicies.yaml
│   │   ├── policy.karmada.io_clusterpropagationpolicies.yaml
│   │   ├── policy.karmada.io_federatedresourcequotas.yaml
│   │   ├── policy.karmada.io_overridepolicies.yaml
│   │   ├── policy.karmada.io_propagationpolicies.yaml
│   │   ├── work.karmada.io_clusterresourcebindings.yaml
│   │   ├── work.karmada.io_resourcebindings.yaml
│   │   └── work.karmada.io_works.yaml
│   ├── kustomization.yaml
│   └── patches
│       ├── webhook_in_clusterresourcebindings.yaml
│       └── webhook_in_resourcebindings.yaml
├── crds.tar.gz
├── karmada-agent.yaml
├── karmada-apiserver.config
├── karmada-scheduler-estimator.yaml
└── pki
    ├── ca.crt
    ├── ca.key
    ├── etcd-ca.crt
    ├── etcd-ca.key
    ├── etcd-client.crt
    ├── etcd-client.key
    ├── etcd-server.crt
    ├── etcd-server.key
    ├── front-proxy-ca.crt
    ├── front-proxy-ca.key
    ├── front-proxy-client.crt
    ├── front-proxy-client.key
    ├── karmada.crt
    └── karmada.key
```

**Which issue(s) this PR fixes**:
Part of #2282 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lonelyCZ** [comment]
Unify format

```
[root@master67 karmada]# tree
.
├── crds.tar.gz
├── karmada-agent.yaml
├── karmada-apiserver.config
├── karmada-scheduler-estimator.yaml
└── pki
    ├── apiserver.crt
    ├── apiserver.key
    ├── ca.crt
    ├── ca.key
    ├── etcd-ca.crt
    ├── etcd-ca.key
    ├── etcd-client.crt
    ├── etcd-client.key
    ├── etcd-server.crt
    ├── etcd-server.key
    ├── front-proxy-ca.crt
    ├── front-proxy-ca.key
    ├── front-proxy-client.crt
    ├── front-proxy-client.key
    ├── karmada.crt
    └── karmada.key
```

File: pkg/karmadactl/cmdinit/kubernetes/deployments.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func (i *CommandInitOption) karmadaAPIServerContainerCommand() []string {
 		fmt.Sprintf("--client-ca-file=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
 		"--enable-admission-plugins=NodeRestriction",
 		"--enable-bootstrap-token-auth=true",
-		fmt.Sprintf("--etcd-cafile=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
+		fmt.Sprintf("--etcd-cafile=%s/%s.crt", karmadaCertsVolumeMountPath, options.EtcdCaCertAndKeyName),
```
Comment (wuyingjun-lucky) [inline pkg/karmadactl/cmdinit/kubernetes/deployments.go:68]: Should we modify in 
`https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/addons/search/manifests.go#:~:text=%2D%20%2D%2Detcd%2Dservers%3D%7B%7B%20.ETCDSevers,pki/karmada.crt` similarly

Comment (lonelyCZ) [reply pkg/karmadactl/cmdinit/kubernetes/deployments.go:70]: Thanks for reminding, fixed.

**Comment by carlory** [comment]
I will review it ASAP this week.

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

/hold 
for @lonelyCZ confirm
Have you tested it?

**Comment by lonelyCZ** [comment]
> Have you tested it?

Yes, it works fine in my environment.

**Comment by RainbowMango** [comment]
Thanks.
/hold cancel

**Pull request merged**